### PR TITLE
11.0 [mig] base_external_dbsource

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by yajo — APPROVED** [review]

File: base_external_dbsource/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
+#. Database sources can be configured in Settings > Configuration ->
+   Data sources.
```
Comment (cubells) [inline base_external_dbsource/readme/CONFIGURE.rst]: Correct menu is `Settings > Technical > Database Structure > Database sources`

File: base_external_dbsource/readme/CREDITS.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[ This file is optional and contains additional credits, other than
+  authors, contributors, and maintainers. ]
+
+The development of this module has been financially supported by:
+
+* Company 1 name
+* Company 2 name
```
Comment (cubells) [inline base_external_dbsource/readme/CREDITS.rst]: I think you can remove this file as It's optional.

File: base_external_dbsource/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+* Go to Settings > Database Structure > Database Sources
```
Comment (cubells) [inline base_external_dbsource/readme/USAGE.rst]: Correct menu is `Settings > Technical > Database Structure > Database sources`

**Review by cubells — APPROVED** [review]
Spanish translation is not completed.
Minor changes.
Code review and test.

**Comment by sergio-teruel** [comment]
@cubells Changes done

**Comment by cubells** [comment]
thanks @sergio-teruel

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
There's a glitch on runbot, but there's no real problem, so I merge.

**Pull request merged**